payment-failure/src/main/scala/com/gu/identity/paymentfailure/abtest/EncryptedEmailTest.scala

```diff
@@ -18,7 +18,7 @@ class EncryptedEmailTest(identityClient: IdentityClient) extends VariantGenerato
       val requestBody = IdentityEmailTokenRequest(email)
       identityClient.encryptEmail(requestBody)
         .bimap(
-          err => new RuntimeException("unable to create email token", err),
+          err => UserIneligibleForAbTest("unable to create email token", Some(err)),
```

Comment:
Same comments as on the `AutoSignInTest` apply. But I wouldn't bother updating this is test in this PR (in fact we could probably get rid of it in another PR).

Reply:
I've added the same logic as Auto Sign In Test for now just for consistency and will make a PR to remove the email test after this has been merged